op-e2e/blobs/blob_test.go

```diff
@@ -0,0 +1,160 @@
+package blobs
+
+import (
+	"bytes"
+	"context"
+	"crypto/ecdsa"
+	"errors"
+	"fmt"
+	"math/big"
+	mrand "math/rand"
+	"testing"
+	"time"
+
+	op_e2e "github.com/ethereum-optimism/optimism/op-e2e"
+	"github.com/ethereum-optimism/optimism/op-e2e/e2eutils/wait"
+	"github.com/ethereum-optimism/optimism/op-e2e/faultproofs"
+	"github.com/ethereum-optimism/optimism/op-e2e/system/e2esys"
+	"github.com/ethereum-optimism/optimism/op-service/eth"
+	"github.com/ethereum-optimism/optimism/op-service/testutils"
+	"github.com/ethereum-optimism/optimism/op-service/txmgr"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/consensus/misc/eip4844"
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethstorage/da-server/pkg/da"
+	"github.com/ethstorage/da-server/pkg/da/client"
+	"github.com/holiman/uint256"
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	dacPort = 37777
+)
+
+var (
+	ctx       = context.Background()
+	dacUrl    = fmt.Sprintf("http://127.0.0.1:%d", dacPort)
+	toAddress = testutils.RandomAddress(mrand.New(mrand.NewSource(dacPort)))
+	blobs     = make([]*eth.Blob, 3)
+)
+
+func TestFunctionSuccess(t *testing.T) {
+	op_e2e.InitParallel(t)
+	dacServer := StartDACServer(t)
+	defer dacServer.Stop(ctx)
+
+	sys, _ := faultproofs.StartFaultDisputeSystem(t, faultproofs.WithDAC(dacUrl))
```

Comment:
Why do we need a dispute system from `faultproofs` here? Is there any relationship between L2 blob and fault proof in this case?

Reply:
no, just reuse this set up func just sgt e2e